pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel", "reentry~=1.3", "fastentrypoints~=0.12"]
```

Comment:
please mention in the eventual commit message that `fastentrypoints` is also being removed

Reply:
just to check, is there a reason you know of why we would still need this? I believe the issue it solved is no longer an issue?

Reply:
This is without fastentrypoints, so I guess it is now ok to remove:

```
#!/Users/chrisjsewell/Documents/GitHub/aiida_core_develop/.tox/py38-pre-commit/bin/python
# EASY-INSTALL-ENTRY-SCRIPT: 'aiida-core','console_scripts','verdi'
import re
import sys

# for compatibility with easy_install; see #2198
__requires__ = 'aiida-core'

try:
    from importlib.metadata import distribution
except ImportError:
    try:
        from importlib_metadata import distribution
    except ImportError:
        from pkg_resources import load_entry_point


def importlib_load_entry_point(spec, group, name):
    dist_name, _, _ = spec.partition('==')
    matches = (
        entry_point
        for entry_point in distribution(dist_name).entry_points
        if entry_point.group == group and entry_point.name == name
    )
    return next(matches).load()


globals().setdefault('load_entry_point', importlib_load_entry_point)


if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(load_entry_point('aiida-core', 'console_scripts', 'verdi')())

```

Reply:
Yeh so actually the current develop `verdi` script is:

```
#!/Users/chrisjsewell/Documents/GitHub/aiida-core-origin/.tox/py39-pre-commit/bin/python
# -*- coding: utf-8 -*-
# EASY-INSTALL-ENTRY-SCRIPT: 'aiida-core','console_scripts','verdi'
__requires__ = 'aiida-core'
import re
import sys

from aiida.cmdline.commands.cmd_verdi import verdi

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(verdi())
```

I guess you need importlib_metadata installed to produce the new script?

Reply:
So I re-added fastentrypoints: I guess that the second script, with the direct `verdi` import, is still sightly quicker